test/SmartEnum.ProtoBufNet.UnitTests/SmartEnumNameSurrogateTests.cs

```diff
@@ -64,7 +64,7 @@ public void SchemaValidation()
         {
             var result = model.GetSchema(typeof(TestEnumBoolean));
 
-            result.Should().BeEquivalentTo(SchemaString);
+            result.Replace("\r\n", "\n").Should().BeEquivalentTo(SchemaString.Replace("\r\n", "\n"));
```

Comment:
Again, I had line ending differences which were failing the tests. There's probably a more elegant way to do this (as I did with json), but I'm not familiar enough with protobuf